### PR TITLE
🛀 Cleanup fields on FlowRun

### PR DESCRIPTION
**Pull request by rowanseymour**
Started adding `FlowRun.status` but changed the `help_text` of something and that seems to be triggering the [Django bug](https://code.djangoproject.com/ticket/25253) which then makes the migration re-add all constraints... so this PR is just the non-database changes in a single migration that we can fake.

### CODE REVIEW

File: locale/es/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -11594,6 +11517,71 @@ msgstr "Elija cuándo le gustaría que empiece el flujo."
 msgid "Facebook Channel"
 msgstr "Borrar teléfono"
 
+#, fuzzy
```
Comment (rowanseymour) [inline locale/es/LC_MESSAGES/django.po]: FYI https://code.djangoproject.com/ticket/10852

File: locale/es/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -3673,83 +3673,6 @@ msgstr "Archivado"
 msgid "User delete"
 msgstr "Rol del usuario"
 
-#, fuzzy
-#| msgid "The call that handled this flow run, only for voice flows"
-msgid "The session that handled this flow run, only for voice flows"
-msgstr "La llamada que manejó esta corrida de flujos, solo para los flujos de voz"
```
Comment (rowanseymour) [inline locale/es/LC_MESSAGES/django.po]: apologies to the person that went to the trouble of translating this 😬

File: temba/flows/models.py [hunk 3]
```diff
@@ -2533,90 +2537,67 @@ class FlowRun(RequireUpdateFieldsMixin, models.Model):
...
     contact = models.ForeignKey(Contact, on_delete=models.PROTECT, related_name="runs")
 
-    session = models.ForeignKey(
-        FlowSession,
-        on_delete=models.PROTECT,
-        related_name="runs",
-        null=True,
-        help_text=_("The session that handled this flow run, only for voice flows"),
-    )
+    # TODO make non-null once we no longer have legacy engine runs
```
Comment (nicpottier) [inline temba/flows/models.py]: Only true if we change to not trimming flow sessions until runs are archived

**Comment by rowanseymour** [comment]
For good measure compared postgresql describe output on the runs table before and after this migration to be 100% sure no actual SQL changes slipped in

File: temba/flows/models.py (hunk 3 above)
Comment (rowanseymour) [reply temba/flows/models.py]: oops good point, I'll drop this comment

**Pull request merged**